public/app/features/explore/state/actions.ts

```diff
@@ -638,7 +633,7 @@ function runQueriesForType(
     const { datasourceInstance, eventBridge, queries, queryIntervals, range, scanning } = getState().explore[exploreId];
     const datasourceId = datasourceInstance.meta.id;
     // Run all queries concurrently
-    const queryPromises = queries.map(async (query, rowIndex) => {
+    queries.map(async (query, rowIndex) => {
```

Comment:
Could be done as for-of since result is not used?

Reply:
Good catch, changed to for loop because we need the rowIndex.